plugins/woocommerce-admin/client/tasks/fills/PaymentGatewaySuggestions/utils.js

```diff
@@ -90,18 +90,7 @@ export const getSplitGateways = (
 	isWCPayOrOtherCategoryDoneSetup
 ) =>
 	Array.from( paymentGateways.values() )
-		.sort( ( a, b ) => {
-			if ( a.hasPlugins === b.hasPlugins ) {
-				return comparePaymentGatewaysByPriority( a, b );
-			}
-
-			// hasPlugins payment first
-			if ( a.hasPlugins ) {
```

Comment:
I removed this because `Affirm` doesn't have "plugins" but it needs to be shown before `Afterpay` for US and CA.